static/app/components/tables/simpleTable/index.tsx

```diff
@@ -24,8 +24,18 @@ export function SimpleTable({children, ...props}: TableProps) {
   );
 }
 
-function Header({children}: {children: React.ReactNode}) {
-  return <StyledPanelHeader role="row">{children}</StyledPanelHeader>;
+function Header({
+  children,
+  ...props
+}: {children: React.ReactNode} & React.DetailedHTMLProps<
```

Comment:
Should we make this an interface and extend instead? I think an intersection with big types hurts tsc perf